3.12-minimal/Dockerfile.fedora

```diff
@@ -0,0 +1,110 @@
+FROM quay.io/fedora/fedora-minimal:39
```

Comment:
I would prefer to use F40 or even F41 if it is possible. F39 will be EOL soon in Dec 2024.